src/services/mongoose/schema.ts

```diff
@@ -285,7 +285,7 @@ UserSchema.virtual("ratioBtc").get(function (this: typeof UserSchema) {
 
 // this is the accounting path in medici for this user
 UserSchema.virtual("walletPath").get(function (this: typeof UserSchema) {
-  return walletPath(this.id)
+  return walletPath(this.walletId)
```

Comment:
Need to dig deeper but IMO this will not work. we still want to use `User.id` because this is used in accounting, and currently `User.id` is what needs to be stored.